src/MatrixFields/single_field_solver.jl

```diff
@@ -50,7 +50,7 @@ single_field_solve!(::ClimaComms.AbstractCPUDevice, cache, x, A, b) =
     _single_field_solve!(cache, x, A, b)
 function single_field_solve!(::ClimaComms.CUDADevice, cache, x, A, b)
     Ni, Nj, _, _, Nh = size(Fields.field_values(A))
-    nthreads, nblocks = Spaces._configure_threadblock(Ni * Nj * Nh)
+    nthreads, nblocks = Topologies._configure_threadblock(Ni * Nj * Nh)
```

Comment:
Can/should we instead move `_configure_threadblock` to `_configure_threadblock` or something?

Reply:
what do you mean?

Reply:
`_configure_threadblock` doesn't really have anything to do with `Topologies`